src/main/java/org/folio/search/cql/CqlSearchQueryConverter.java

```diff
@@ -251,8 +251,8 @@ private String getSearchTerm(String term, String fieldName, String resource) {
 
   private List<String> getFieldsForMultilangField(String fieldName, String resource) {
```

Comment:
Why .filter(PlainFieldDescription::isFulltex  is used in getFieldsForMultilangField?